gammapy/cube/basic_cube.py

```diff
@@ -0,0 +1,74 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Functions to perform basic functions for map and cube analysis.
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+from ..maps import WcsNDMap
+
+
+def fill_map_counts(event_list, ndmap):
+    """Fill a ``WcsMap` with events from an EventList.
+
+     The energy of the events is used for a non-spatial axis homogeneous to energy.
+     The other non-spatial axis names should have an entry in the colum names of the ``EventList``
+
+     Parameters
+     ----------
+     event_list : `~gammapy.data.EventList`
+             the input event list
+     ndmap : `~gammapy.maps.Map`
+         Target map
+     """
+    # The list will contain the event table entries to be fed into the WcsNDMap
+    coord_dict = dict()
+    ref_geom = ndmap.geom
+
+    # Add sky coordinates to dictionary
+    coord_dict.update(skycoord=event_list.radec)
+
+    # Now check the other axes and find corresponding entries in the EventList
+    # energy and time are specific types
+    # TODO: add proper extraction for time
```

Comment:
Move this TODO down below the `energy` handling? That would be the location where to put the time handling special code, no?